rpc-services/qubesbuilder.TriggerBuild

```diff
@@ -48,6 +49,10 @@ while read -r release_name builder_dir; do
         continue;
     fi
 
+    if ! [ -d "$builder_dir/qubes-src/$untrusted_component_name" ]; then
```

Comment:
This is very wrong. First of all, this breaks building a component freshly added to the config. But even if not that, there are cases where component name does not match repository name (for example when several branches of the same repo are build - like for the kernel). The loop just below is validating it the proper way: by getting repository name for each component and comparing that.

Reply:
yeah, this is wrong.